ext/crypto/lib.rs

```diff
@@ -1652,18 +1652,31 @@ pub fn op_crypto_random_uuid(
 
 pub async fn op_crypto_subtle_digest(
   _state: Rc<RefCell<OpState>>,
-  algorithm: CryptoHash,
+  algorithm: String,
   data: ZeroCopyBuf,
 ) -> Result<ZeroCopyBuf, AnyError> {
-  let output = tokio::task::spawn_blocking(move || {
-    digest::digest(algorithm.into(), &data)
-      .as_ref()
-      .to_vec()
-      .into()
-  })
-  .await?;
-
-  Ok(output)
+  let output = if algorithm == "MD-5" {
+    tokio::task::spawn_blocking(move || md5::compute(&data).to_vec().into())
+      .await
+  } else {
+    let algorithm = match algorithm.as_str() {
+      "SHA-1" => CryptoHash::Sha1,
+      "SHA-256" => CryptoHash::Sha256,
+      "SHA-384" => CryptoHash::Sha384,
+      "SHA-512" => CryptoHash::Sha512,
+      _ => return Err(not_supported()),
+    };
```

Comment:
It'd be more elegant to deserialize to CryptoHash here it's also more complicated and probably means having ext/crypto depend on serde_v8.